### DISCUSSION TITLE
How do I make a Sprite stay on one spot on the screen in make a code arcade

### DISCUSSION THREAD

**Discussion by Kitty3119** [community/community#129014] [Programming Help] [answered]
### Body

I am currently making a game on make a code arcade but I have a problem. I need a sprite to stay onthe screen and in a specific space (it's a button). Also it shouldn't be affected by the tile map. Does anyone know how to do that and if so please help. Thanks.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Kitty3119** [comment, marked as answer]
Nevermind, I figured out a way : )
If you are wondering this is what you can do
![2024-06-19 (1)](https://github.com/community/community/assets/173133036/55a2ca20-5475-49f1-a261-b097f4f02c96)

**Reply by ebndev** [reply]
@Kitty3119 thanks for being a part of the GitHub Community! 

We appreciate you taking the time to come back and post the solution for future users to find 😄
